core/lib/contract_verifier/src/lib.rs

```diff
@@ -556,7 +557,33 @@ impl ContractVerifier {
         request_id: usize,
         contract: &DeployedContractData,
         creation_bytecode: &[u8],
+        contract_identifier: &ContractIdentifier,
     ) -> Result<ConstructorArgs, ContractVerifierError> {
+        fn extract_arguments<'a>(
```

Comment:
Nit: IMO, it'd be more readable to place this function on the `decode_evm_constructor_args()` level.

Reply:
I would prefer to keep it here -- it feels too low-level (including `&'static str` as an `Err` type) to exist outside single scope.